hack/lib/util/environment.sh

```diff
@@ -145,12 +145,14 @@ function os::util::environment::setup_tmpdir_vars() {
     HOME="${FAKE_HOME_DIR}"
     export HOME
 
-    # ensure that the directories are clean
-    for target in $( ${USE_SUDO:+sudo} findmnt --output TARGET --list ); do
-        if [[ "${target}" == "${BASETMPDIR}"* ]]; then
-            ${USE_SUDO:+sudo} umount "${target}"
-        fi
-    done
+    if [ "$(uname)" != "Darwin" ]; then
```

Comment:
If we are going to just disable this for Darwin, some comments:

Prefer `[[` over `[`:
```sh
if [[ "$(uname)" != "Darwin" ]]; then
```

Indent with tabs.